custom-glossary.hbs

```diff
@@ -0,0 +1,76 @@
+{{!< default}}
+
+<main class="site__content">
+
+    {{#post}}
+        <article class="article">
+
+            <header class="section section--dense background-alternate">
+                <div class="container container--article">
+
+                    <div class="article__meta">
+                        {{tags}}
+                    </div>
+
+                    <h1 class="article__title js-nbsp">{{title}}</h1>
+                    <h2 class="article__summary js-nbsp">{{excerpt}}</h2>
+
+                </div>
+            </header>
+
+            <div class="section section--dense-top">
+                <div class="container container--article">
+
+                    <div class="editor js-nbsp js-external-links">
+                        {{content}}
+                    </div>
+
+                    <hr class="spacing-bottom-none" />
+
+                </div>
+            </div>
+
+            <footer class="section section--dense">
+                <div class="container container--article">
+
+                    <p class="text-small-block spacing-bottom-none">
+                        <em>
```

Comment:
Si skoro říkám, jestli má smysl ten text zabalit do `em`, když styl kurzíva lze aplikovat už na `p`, ale to je marginální poznámka.

Reply:
Máš pravdu, sémanticky tu `em` nemá význam. Třídu pro kurzívu jsem tedy vytvořil.